packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -203,78 +204,47 @@ const EDiscovery = WebexPlugin.extend({
   },
 
   @waitForValue('@')
-  @oneFlight({keyFactory: (reportId) => `${reportId}`})
+  @oneFlight({keyFactory: (reportId, options) => String(reportId + (options && options.offset && options.size ? options.offset + options.size : ''))})
```

Comment:
Because this is used multiple times and use of the offset is important to how this works, it would be good to break this function out and document it with JSDoc.